### ISSUE TITLE
Packages with load errors

### ISSUE THREAD

**Issue by gdkrmr** [tonyhffong/Lint.jl#233]
If there is a Package that has `LoadError` JSON output may not be correct in the lint server:
See here: https://github.com/gdkrmr/flycheck-julia/issues/1
and maybe here: https://github.com/TeroFrondelius/linter-julia/issues/43

I also noticed that packages with `LoadError`s are very bad for performance.

**Comment by TotalVerb** [comment]
Thanks for the report. Could you try `Pkg.checkout("Lint")` first, and see if the problem persists?

**Comment by gdkrmr** [comment]
still persists

**Comment by TotalVerb** [comment]
Thanks, the version is as expected. I will try to reproduce the issue. Is CUDArt the one that causes it?

**Comment by gdkrmr** [comment]
yes, the problem is that it does not work and calling `using CUDArt` produces error messages on my machine.

**Comment by gdkrmr** [comment]
can you reproduce this? if you do not have an nvidia card then CUDArt should give you the same error when you try to build it.

**Comment by TotalVerb** [comment]
I cannot reproduce the slowness. I will investigate the lintserver issue sometime soon.

**Comment by TotalVerb** [comment]
@gdkrmr I was unable to reproduce the issue with CUDArt on master. Lint correctly reported that the dynamic import failed and it could not determine the symbols exported.
